crates/uv-cli/src/lib.rs

```diff
@@ -4290,3 +4293,65 @@ pub struct DisplayTreeArgs {
     #[arg(long, alias = "reverse")]
     pub invert: bool,
 }
+
+#[derive(Args, Debug)]
+pub struct PublishArgs {
+    /// The paths to the files to uploads, as glob expressions.
+    #[arg(default_value = "dist/*")]
```

Comment:
Could we document this in the long help? Do we need to change this to only glob sdists and wheels?

Reply:
We're already ignoring files that are neither source dists nor wheels

Reply:
We should mention that in the reference documentation here.

Reply:
Good point, added